docs/topics/js/js-project-setup.md

```diff
@@ -2,8 +2,10 @@
 
 Kotlin/JS projects use Gradle as a build system. To let developers easily manage their Kotlin/JS projects, we offer
 the `kotlin.multiplatform` Gradle plugin that provides project configuration tools together with helper tasks for automating routines
-typical for JavaScript development. For example, the plugin downloads the [Yarn](https://yarnpkg.com/) package manager
-for managing [npm](https://www.npmjs.com/) dependencies in background and can build a JavaScript bundle from a Kotlin project
+typical for JavaScript development.
+
+The plugin uses the [npm](https://www.npmjs.com/) or [Yarn](https://yarnpkg.com/) package manager
```

Comment:
"uses ... to download ... and can build" can be a bit confusing.

I'd alter the sentence structure to be more uniform, for example: "The plugin downloads dependencies using ... and builds JavaScript bundles from Kotlin projects using ..."